src/Elmish.WPF/ViewModel.fs

```diff
@@ -357,25 +348,22 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
     | Cmd _
     | CmdParam _ ->
         false
-    | SubModel ((vm: ViewModel<obj, obj> voption ref), (getModel: 'model -> obj voption), getBindings, toMsg, sticky) ->
```

Comment:
Can't remember why I put these annotations there in the first place, so please just make sure that this actually is behaviour preserving (not everything is with weakly typed stuff like this, even if it compiles).

Reply:
That someone would be me, and you're probably right. But there are cases in this file of type annotations being required even when it doesn't seem it should. So it might be that it was needed earlier on, but not after some refactor.